nextcloudappstore/core/templates/nav.html

```diff
@@ -3,6 +3,20 @@
 <div class="banner">
     <div class="container">
         <a class="brand" href="/"><img class="logo" src="{% static 'assets/img/logo_nextcloud_white.svg' %}" title="nextcloud.com" itemprop="logo"></a>
+        <form method="get"
+              action="{% if current_category %}{{ request.path }}
+                      {% else %}{% url 'home' %}
+                      {% endif %}"
+              class="search-form">
+            <input type="text"
+                   name="search"
+                   value="{{ search_query }}"
+                   placeholder="{% if current_category %}Search in {{ current_category.name }}
```

Comment:
I'd probably drop the in: {% if current_category %}Search {{ current_category.name }}

Also needs blocktrans